Foundation/FileManager.swift

```diff
@@ -530,22 +530,14 @@ open class FileManager : NSObject {
         }
     }
 
-    open func copyItem(atPath srcPath: String, toPath dstPath: String) throws {
+    private func _processItem(atPath srcPath: String, toPath dstPath: String, _ body: (String, String, FileAttributeType) throws -> ()) throws {
```

Comment:
This method seems too generically named for something that creates directories. The rest looks great though.